libr/core/panels.c

```diff
@@ -4889,7 +4889,7 @@ static RPanels *get_cur_panels(RPanelsRoot *panels_root) {
 	return get_panels (panels_root, panels_root->cur_panels);
 }
 
-static bool handle_tab(RCore *core) {
+static bool handle_tab(RCore *core, bool *del_tab) {
```

Comment:
Not a big fan of this new signature. It doesnt escales and its confusing to use. Im fine for merging i guess you’ll be diing more refactors over time

Reply:
yup exactly. i will be refactoring this later on but currently '-' keys in tab emits a bug anyway without this.